patch/attiny861.yaml

```diff
@@ -0,0 +1,26 @@
+_svd: ../svd/attiny861.svd
+
+_include:
+  - "common/adc.yaml"
+  - "common/port.yaml"
+  - "common/tiny/usi.yaml"
+  - "common/wdt.yaml"
+
+  - "timer/attiny861.yaml"
+
+AC:
+  _modify:
+    ACSRA:
+      access: read-write
+  ACSRA:
+    _modify:
+      ACIS:
+        description: "Analog Comparator Interrupt Mode Select"
+      ACO:
+        access: read-only
+    ACIS:
+      _replace_enum:
+        ON_TOGGLE: [0, "Interrupt on Toggle"]
+        # Leaving [1, 'Reserved'] out
+        ON_FALLING_EDGE: [2, "Interrupt on Falling Edge"]
+        ON_RISING_EDGE: [3, "Interrupt on Rising Edge"]
```

Comment:
Similar change to above should work here as well I think.

Reply:
When I added this change, it broke the AC patch for atmega328pb, which I think is because of an error in the svd file for that chip. According to the svd file, there's a register called ACSRA with no fields (in addition to the ACSR register with the ACIS field), but the ACSRA field doesn't show up at all in the datasheet as far as I can tell. For now, I made the changes to the common AC patch, and replaced the common patch in atmega328pb.yaml with one that works. I'm not sure this is the best solution though.